google/detectors/credentials/generic_weak_credential_detector/src/test/resources/com/google/tsunami/plugins/detectors/credentials/genericweakcredentialdetector/testers/grafana/testdata/dashboardsPage.html

```diff
@@ -0,0 +1,212 @@
+        themePaths: {
+          light: 'public/build/grafana.light.bbe69ddb3979b7904078.css',
+          dark: 'public/build/grafana.dark.170498723865582cad9d.css'
+        }
+      };
+
+
+
+      if (window.grafanaBootData.user.theme === "system") {
+        document.body.classList.remove("theme-system");
+        var darkQuery = window.matchMedia("(prefers-color-scheme: dark)");
+        var cssLink = document.createElement("link");
+        cssLink.rel = 'stylesheet';
+
+        if (darkQuery.matches) {
+          document.body.classList.add("theme-dark");
+          cssLink.href = window.grafanaBootData.themePaths.dark;
+          window.grafanaBootData.user.lightTheme = false;
+        } else {
+          document.body.classList.add("theme-light");
+          cssLink.href = window.grafanaBootData.themePaths.light;
+          window.grafanaBootData.user.lightTheme = true;
+        }
+        document.head.appendChild(cssLink);
+      }
+
+      window.__grafana_load_failed = function() {
+        var preloader = document.getElementsByClassName("preloader");
+        if (preloader.length) {
+          preloader[0].className = "preloader preloader--done";
+        }
+      }
+
+
+      window.onload = function() {
+        if (window.__grafana_app_bundle_loaded) {
+          return;
+        }
+        window.__grafana_load_failed();
+      };
+
+
+      </script><script nonce="" src="public/build/runtime.3351b7db5d116965e49d.js"></script><script nonce="" src="public/build/147.168f5921b2ea9d44f496.js"></script><script nonce="" src="public/build/4832.e9597303bdeccac41938.js"></script><script nonce="" src="public/build/1537.193988337753ed98ad56.js"></script><script nonce="" src="public/build/4019.50ac816d6b33311ef7c8.js"></script><script nonce="" src="public/build/app.bba965d0c95be02c72d0.js"></script><script nonce="">performance.mark('frontend_boot_js_done_time_seconds');</script></body></html>
```

Comment:
Lint: This file is missing a terminating newline. Could you please add it?

Reply:
This file was replaced by a new one with a terminating newline